roles/tpa_single_node/templates/manifests/guac/graphql/Deployment.yaml

```diff
@@ -41,10 +41,12 @@ spec:
                   '
           env:
             - name: TC_PGHOST
-              valueFrom:
-                secretKeyRef:
-                  key: POSTGRES_HOST
-                  name: keycloak-db-secret
+              value: keycloak-postgresql
```

Comment:
I see this is WIP, But just to confirm - dont we have a secret resource with "POSTGRES_HOST"?

Reply:
That's right, we don't have that yet.